virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar/launchbar.component.html

```diff
@@ -10,9 +10,12 @@
   Copyright Contributors to the Zowe Project.
 -->
 
-<div class="launchbar-container launchbar-large" id="container"
-     [style.height]="barSize"
-     (mouseup)="onMouseUpContainer($event)" [class.active]="isActive">
+<div class="launchbar-container launchbar-large" 
+  id="container"
```

Comment:
why do we need `id` here?

Reply:
Just for formatting because it looked wonky

Reply:
Hate having to scroll along the y axis to see something, maybe that's just personal preference

Reply:
Why do we need `id` at all? Do we reference it anywhere?

Reply:
Probably QA/automation. It was there so I won't touch it for now